src/compare-oracle-to-fallback/compare-oracle-to-fallback.js

```diff
@@ -0,0 +1,185 @@
+const ethers = require('ethers');
+const BigNumber = require('bignumber.js');
+const {
+  Finding, FindingSeverity, FindingType, getJsonRpcUrl,
+} = require('forta-agent');
+
+// load required shared types
+const {
+  LendingPoolAddressesProvider: lendingPoolAddressesProvider,
+  ProtocolDataProvider: protocolDataProviderAddress,
+} = require('../../contract-addresses.json');
+const { abi: protocolDataProviderAbi } = require('../../interfaces/AaveProtocolDataProvider.json');
```

Comment:
interfaces -> abi

Reply:
Replaced all reference to `interfaces` with `abi`.